dev_requirements.txt

```diff
@@ -1,6 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt.git#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@ct-236-adapter_tests#egg=dbt-core&subdirectory=core
```

Comment:
Is this temporary? Are we planning not to merge this until the corresponding fixtures are merged into Core?